completions-compute.txt

```diff
@@ -12,6 +12,7 @@ ConversionType
 DivideBy
 EntryKey
 ExcludeRegExp
+Json2Csv
```

Comment:
Please remove. This is only for keywords **after** `Compute(n).`, like *Column*, *TranslationTable*, or in the case of *XML2CSV*: *EntryKey*, etc.